.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@ Have any questions? Check out the contributing docs at https://github.com/ovh/ma
 
 | Question         | Answer
 | ---------------- | ---
-| Branch?          | `master` for hotfix / `develop` for bug fixes <!-- target branch -->
+| Branch?          | `master` for hotfixes / `release/**` / `develop` for bug fixes  <!-- target branch -->
```

Comment:
**nitpick**
Does it makes sense to keep speaking about fixes here? As features may also target those branches